includes/GbWeiss.php

```diff
@@ -80,6 +76,11 @@ public function initOptionPage(): void
             ->addOption(new Option('Client Secret', 'client_secret', __('Client Secret', self::$languageDomain), 'account', 'string'));
 
         $optionsPage->addTab($accountTab);
+        $orderStatuses = $this->orderStateRepository->getAllOrderStates();
```

Comment:
i'd write orderStates

Reply:
To my knowledge the singular entity name is used when following the repository pattern.

Reply:
but the term "statues" does not exist in the language. could be "status" or "stati" ... but this is "haarspalterei" ;)

Reply:
The name actually comes from WooCommerce, but I'll change it.